src/Grpc.Core.Api/Status.cs

```diff
@@ -88,7 +88,8 @@ public override string ToString()
     {
         if (DebugException != null)
         {
-            return $"Status(StatusCode=\"{StatusCode}\", Detail=\"{Detail}\", DebugException=\"{DebugException}\")";
+            return $"Status(StatusCode=\"{StatusCode}\", Detail=\"{Detail}\"," +
+                $" DebugException=\"{DebugException.GetType().Name}: {DebugException.Message}\")";
```

Comment:
`Exception.ToString()` includes the namespace of the exception (`InvalidOperationException` vs `System.InvalidOperationException`).

The goal is to replicate ToString, but without the long stack trace. This is slightly more correct:
```suggestion
                $" DebugException=\"{DebugException.GetType()}: {DebugException.Message}\")";
```